### ISSUE TITLE
make error ---Headache problem ina long time

### ISSUE THREAD

**Issue by zllshu** [liulei01/DRBox#33]
![default](https://user-images.githubusercontent.com/36331213/50902948-5c9cc900-1457-11e9-94eb-9ac4d4479a09.PNG)
it's a strange question while I make the project.I spent a lot of time debugging。But there are almost no corresponding answers to the online answers.So I have to ask you for help, can you give me some advice?

**Comment by yaceben** [comment]
Do you have libqt5 or something installed (im not on ubuntu but should be easy enough to find...)?

**Comment by zllshu** [comment]
> Do you have libqt5 or something installed (im not on ubuntu but should be easy enough to find...)?
Yeah,actually I find libqt5 in usr/lib/ ,but when i try to use command **ldd libcaffe.so.1.0.0-rc3 |grep opencv**,then i get the following :
![1](https://user-images.githubusercontent.com/36331213/50942248-f3529f80-14c2-11e9-8c48-9d25cc1d899e.PNG)
but  when  i open libQT5IopenGL.so, i cant't find reference to qt_5

**Comment by yaceben** [comment]
Never encountered this, but from https://stackoverflow.com/questions/36128645/error-on-execution-version-qt-5-not-found-required-by
I would suggest trying first 

> 
`sudo apt-get install qt5-default`
> 

and if it still not working, force point your execution by prepending LD_LIBRARY_PATH  as such

> Issue happens because you have Qt installed in /usr/lib/x86_64-linux-gnu from distro which ld will use by default. To avoid such issues you can compile with -rpath or **run with LD_LIBRARY_PATH=/path/to/your/Qt/libs/location to help linker find needed Qt version**.

eg. `LD_LIBRARY_PATH=/path/to/your/Qt/libs/location make -j<nthreads> ...`

or fiddle a bit in the makefiles to point it to the correct qt lib paths

Good luck!

**Comment by zllshu** [comment]
Thank you very much for your kindness, I will try your method later.

**Comment by zllshu** [comment]
> Never encountered this, but from https://stackoverflow.com/questions/36128645/error-on-execution-version-qt-5-not-found-required-by
> I would suggest trying first
> 
> > 
> 
> `sudo apt-get install qt5-default`
> 
> > 
> 
> and if it still not working, force point your execution by prepending LD_LIBRARY_PATH as such
> 
> > Issue happens because you have Qt installed in /usr/lib/x86_64-linux-gnu from distro which ld will use by default. To avoid such issues you can compile with -rpath or **run with LD_LIBRARY_PATH=/path/to/your/Qt/libs/location to help linker find needed Qt version**.
> 
> eg. `LD_LIBRARY_PATH=/path/to/your/Qt/libs/location make -j<nthreads> ...`
> 
> or fiddle a bit in the makefiles to point it to the correct qt lib paths
> 
> Good luck!

it's bad news for me,it don't work for my work any more
